plans/phase-1/final-cr-definition.md

```diff
@@ -0,0 +1,845 @@
+  # This is always populated by the parent controller.
+  globalTopoServer:
+    rootPath: "/multigres/global"
+    clientServiceName: "example-multigres-cluster-global-client"
+
+  # ALTERNATIVE CONFIG: (Using the Global TopoServer)
+  #
+  # Because the 'us-east-1' cell in the parent CR had no 'topoServer'
+  # block, the MultigresCluster controller sets this to 'global'.
+  # The Cell controller will use this to configure its MultiGateway
+  # to talk to the global topo server.
+  topoServer:
+    global:
+      rootPath: "/multigres/global"
+
+  # ALTERNATIVE CONFIG: Inline Definition (External)
+  # If this were 'us-east-1', the MultigresCluster controller would
+  # have copied the 'external' block directly, like this:
+  #
+  # topoServer:
+  #   external:
+  #     address: "etcd-us-east-1.my-domain.com:2379"
+  #     rootPath: "/multigres/us-east-1"
+
+  # ALTERNATIVE CONFIG: (Managed Local) ---
+  # If this were 'us-west-2', the MultigresCluster controller would
+  # have copied the 'managedSpec' block directly, like this.
+  # The Cell controller would then be responsible for creating
+  # a NEW 'TopoServer' CR from this spec, with this Cell
+  # as its owner.
+  #
+  # topoServer:
+  #   managedSpec:
+  #     rootPath: "/multigres/us-west-2"
+  #     image: "quay.io/coreos/etcd:v3.5.17"
+  #     replicas: 1
+  #     dataVolumeClaimTemplate:
+  #       accessModes: ["ReadWriteOnce"]
+  #       resources:
+  #         requests:
+  #           storage: "5Gi"
+
+  # List of all cells in the cluster for discovery.
+  allCells:
+  - "us-east-1"
+  - "us-west-2"
+  - "eu-central-1"
+
+  # Topology flags for the Cell controller to act on.
+  topologyReconciliation:
+    registerCell: true
+    pruneTablets: true
+status:
+  conditions:
+  - type: Available
+    status: "True"
+    lastTransitionTime: "2025-10-21T10:36:00Z"
+    message: "MultiGateway is healthy"
+  gatewayReplicas: 2
+  gatewayReadyReplicas: 2
+  gatewayServiceName: "example-multigres-cluster-us-east-1-gateway"
+  multiorchAvailable: "True"
+  ```
+
+  ## TableGroup CR - Read-Only Child of MultigresCluster
+
+```yaml
+# This child CR is created from the 'orders_tg' entry
+# under 'production_db' in the MultigresCluster CR.
+#
+# The MultigresCluster controller does the following:
+# 1. Copies the *relevant* images (postgres, multipooler) into `spec.images`.
+# 2. The multigresCluster controller watches and resolves the `deploymentTemplate` entries into the 'shardTemplate'.
+#
+apiVersion: multigres.com/v1alpha1
+kind: TableGroup
+metadata:
+  # Name is derived from database + table group name
+  name: "production-db-orders-tg"
+  namespace: multigres
+  creationTimestamp: "2025-10-21T10:30:02Z"
+  generation: 1
+  resourceVersion: "12347"
+  uid: "d4e5f6a7-1234-5678-90ab-f0e1d2c3b4a7"
+  labels:
+    multigres.com/cluster: "example-multigres-cluster"
+    multigres.com/database: "production_db"
+    multigres.com/table-group: "orders_tg"
```

Comment:
The same discussion as Cell, I'm not sure why we need the table-group label for the table-group resource itself.
Actually, there is no other place that has "orders_tg" in the entire CR, other than the metadata.name containing both database name + table group name.

Is it maybe OK to drop "database" label as well, because the resource name already has that detail? (Any child CRs and components of TableGroup should have this label populated, though.)